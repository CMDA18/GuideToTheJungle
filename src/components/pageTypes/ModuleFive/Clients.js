// @flow
import React, { Component } from 'react'
import Helmet from 'react-helmet'
// import styled from 'styled-components'
import Grid from '../../Grid'
import PageTitle from '../../PageTitle'
import Intro from '../../Intro'
import { clients } from '../../../constants/content/index'
import { logos } from '../../../constants/content/logos'
import LogosCollection from '../../LogosCollection'
import Task from '../../Task'

type Props = {
  test: string
}

class Clients extends Component<Props> {
  render () {
    return (
      <div className='clients' data-test={this.props.test}>
        <Helmet
          title='clients'
        />
        <Grid page='ourClients'>
          <header>
            <PageTitle {...clients}/>
            <Intro {...clients}/>
          </header>
          <LogosCollection {...logos}/>
          <Task {...clients}/>
        </Grid>
      </div>
    )
  }
}

export default Clients
