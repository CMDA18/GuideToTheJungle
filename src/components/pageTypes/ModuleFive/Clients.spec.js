import React from 'react'
import { shallow } from 'enzyme'
import Clients from './Clients'

it('renders without crashing', () => {
  expect(shallow(<Clients />)).toMatchSnapshot()
})
