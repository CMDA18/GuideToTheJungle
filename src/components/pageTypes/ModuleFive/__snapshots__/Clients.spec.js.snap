// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Clients />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <HelmetWrapper
          defer={true}
          encodeSpecialCharacters={true}
          title="clients"
        />,
        <Grid
          page="ourClients"
        >
          <header>
            <PageTitle
              introText="Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven."
              introTitle="Onze evolutie in tijd"
              pageTitle="5.1 Klanten"
            />
            <Intro
              introText="Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven."
              introTitle="Onze evolutie in tijd"
              pageTitle="5.1 Klanten"
            />
          </header>
        </Grid>,
      ],
      "className": "clients",
      "data-test": undefined,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "defer": true,
          "encodeSpecialCharacters": true,
          "title": "clients",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <header>
            <PageTitle
              introText="Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven."
              introTitle="Onze evolutie in tijd"
              pageTitle="5.1 Klanten"
            />
            <Intro
              introText="Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven."
              introTitle="Onze evolutie in tijd"
              pageTitle="5.1 Klanten"
            />
          </header>,
          "page": "ourClients",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <PageTitle
                introText="Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven."
                introTitle="Onze evolutie in tijd"
                pageTitle="5.1 Klanten"
              />,
              <Intro
                introText="Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven."
                introTitle="Onze evolutie in tijd"
                pageTitle="5.1 Klanten"
              />,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "introText": "Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven.",
                "introTitle": "Onze evolutie in tijd",
                "pageTitle": "5.1 Klanten",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "introText": "Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven.",
                "introTitle": "Onze evolutie in tijd",
                "pageTitle": "5.1 Klanten",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "header",
        },
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <HelmetWrapper
            defer={true}
            encodeSpecialCharacters={true}
            title="clients"
          />,
          <Grid
            page="ourClients"
          >
            <header>
              <PageTitle
                introText="Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven."
                introTitle="Onze evolutie in tijd"
                pageTitle="5.1 Klanten"
              />
              <Intro
                introText="Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven."
                introTitle="Onze evolutie in tijd"
                pageTitle="5.1 Klanten"
              />
            </header>
          </Grid>,
        ],
        "className": "clients",
        "data-test": undefined,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "defer": true,
            "encodeSpecialCharacters": true,
            "title": "clients",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <header>
              <PageTitle
                introText="Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven."
                introTitle="Onze evolutie in tijd"
                pageTitle="5.1 Klanten"
              />
              <Intro
                introText="Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven."
                introTitle="Onze evolutie in tijd"
                pageTitle="5.1 Klanten"
              />
            </header>,
            "page": "ourClients",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <PageTitle
                  introText="Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven."
                  introTitle="Onze evolutie in tijd"
                  pageTitle="5.1 Klanten"
                />,
                <Intro
                  introText="Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven."
                  introTitle="Onze evolutie in tijd"
                  pageTitle="5.1 Klanten"
                />,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "introText": "Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven.",
                  "introTitle": "Onze evolutie in tijd",
                  "pageTitle": "5.1 Klanten",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "introText": "Ons bedrijf is in 2000 opgericht als het eerste digital focussed onderzoeks- en strategiebureau van Nederland. Sindsdien blijven we deze titel verdienen door producten te maken die er echt toe doen in iemands leven.",
                  "introTitle": "Onze evolutie in tijd",
                  "pageTitle": "5.1 Klanten",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "header",
          },
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
